plugins/auto-sizes/hooks.php

```diff
@@ -186,7 +186,15 @@ function auto_sizes_improve_image_sizes_attributes( string $content ): string {
 			}
 			break;
 
-		// @todo: handle left/right alignments.
+		case 'left':
+		case 'right':
+		case 'center':
+			// Resize image width.
+			$image_width = $processor->get_attribute( 'data-resize-width' ) ?? $image_width;
+			$width       = auto_sizes_get_width( '', (int) $image_width );
```

Comment:
I'm looking at this function:

https://github.com/WordPress/performance/blob/892b0dc7cce7614e9e17ccf23f4488dec6c38359/plugins/auto-sizes/hooks.php#L106-L111

When the first argument doesn't end in `px` it then appends `px`. Therefore, why wouldn't this just be:

```suggestion
			$width       = sprintf( '%dpx', $image_width );
```

Reply:
Nice catch.